test/components/Settings/helper.spec.js

```diff
@@ -18,6 +21,10 @@ describe('Settings Helper', () => {
     const fakeRequest2 = {
       fetchStatus: 'loaded'
     }
-    expect(isFetching([fakeRequest1, fakeRequest2])).toBe(false)
+    expect(isFetchingQueries([fakeRequest1, fakeRequest2])).toBe(false)
+  })
+
+  it('should return true for cozyClientCanCheckPremium', () => {
```

Comment:
I think this test should be clearer if cozyClientCanCheckPremium would take CozyClient in argument. This way you could test both true/false behaviors.

Reply:
Thanks :)